libp2p/switch.nim

```diff
@@ -295,9 +298,6 @@ proc stop*(s: Switch) {.async, public.} =
     except CatchableError as exc:
       warn "error cleaning up transports", msg = exc.msg
 
-  for service in s.services:
```

Comment:
Could you please mention this in the description?

Reply:
done